frontend/src/types.d.ts

```diff
@@ -0,0 +1,12 @@
+// declaring module will allow typescript to import the module
+declare module 'react-bootstrap-table-next' {
+  // typing module default export as `any` will allow you to access its members without compiler warning
+  var bootstrap: any; 
+  export default bootstrap;
+}
+
+declare module 'react-bootstrap-table2-editor' {
+  // typing module default export as `any` will allow you to access its members without compiler warning
+  var bootstrap: any; 
+  export default bootstrap;
+}
```

Comment:
This is how you can hack modules that aren't supported by typescript (no types defined).  I couldn't find any for these two libraries so I just added this and stuff just works.